src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.tsx

```diff
@@ -1,55 +1,125 @@
-import React, { useCallback, useState } from "react";
+import React, { useCallback } from "react";
 import { Modal } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+import { MAX_TABLE_CAPACITY, MIN_TABLE_CAPACITY } from "settings";
+
+import { InputField } from "components/atoms/InputField";
 
 import "./EditTableTitleModal.scss";
 
 export interface EditTableTitleModalProps {
   isShown: boolean;
   title: string;
+  description?: string;
+  capacity: number;
   error: string;
   onHide: () => void;
   onCancel: () => void;
-  onSave: (newTableTitle: string) => void;
+  onSave: (values: EditTableForm) => void;
+}
+
+export interface EditTableForm {
+  title: string;
+  description?: string;
+  capacity: number;
 }
 
 export const EditTableTitleModal: React.FC<EditTableTitleModalProps> = ({
   isShown,
   title,
+  description,
+  capacity,
   error,
   onSave,
   onHide,
   onCancel,
 }) => {
-  const [newTableTitle, setNewTableTitle] = useState<string>(title);
+  const {
+    register,
+    handleSubmit,
+    errors,
+    // formState,
+    // setError,
+    // clearError,
```

Comment:
Commented?

Reply:
Removed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a